lms/services/blackboard_api/_basic.py

```diff
@@ -106,9 +106,11 @@ def _send(self, method, url):
         try:
             return self._oauth_http_service.request(method, url)
         except ExternalRequestError as err:
+            err.refreshable = getattr(err.response, "status_code", None) == 401
```

Comment:
Oh I read this wrong in the first pass. This err would affect both the "Bearer token is invalid" case and the plain `raise` one. :+1: